docs/_tutorials/accelerator-setup-guide.md

```diff
@@ -8,6 +8,7 @@ tags: getting-started
 - [Introduction](#introduction)
 - [Intel Architecture (IA) CPU](#intel-architecture-ia-cpu)
 - [Intel XPU](#intel-xpu)
+- [NPU](#npu)
```

Comment:
Do you want to add Huawei here?

Reply:
I think it's better to change NPU to Huawei Ascend NPU here. @xuedinge233

Reply:
Ok, I have finished the modification